src/Appwrite/Promises/Swoole.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Appwrite\Promises;
+
+use Swoole\Coroutine\Channel;
+
+class Swoole extends Promise
+{
```

Comment:
What is this class responsible for? why is it part of the promise namespace?

Reply:
This extends the promise base, using a swoole coroutine to execute the promise. Akin to extending the base adapter for a vendor specific implementation